2022-08-24-Pods-BlogPost/sample_ml-model/train.py

```diff
@@ -28,11 +30,17 @@ def handler(event, context):
     # Learn the digits on the train subset
     clf.fit(X_train, y_train)
 
-    # Predict the value of the digit on the test subset
-    predicted = clf.predict(X_test)
-    print("--> prediction result:", predicted)
-
-    # Note: the trained model can be used here for predictions, uploaded the S3 again, etc...
+    # Dump the trained model to S3
+    s3_client = boto3.client("s3")
+    with tempfile.TemporaryFile() as tf:
+        dump(clf, tf)
```

Comment:
nit: haven't tested this myself, but we should be able to handle the dump in-memory (without external tmp file) using `io.BytesIO` - something like:
```
    buffer = io.BytesIO()
    dump(clf, buffer)
    s3_client.put_object(Body=buffer.getvalue(), Bucket="pods-test", Key="model.joblib")
```